mobile/lib/features/trade/position_change_notifier.dart

```diff
@@ -30,6 +36,9 @@ class PositionChangeNotifier extends ChangeNotifier implements Subscriber {
 
     if (event is bridge.Event_PositionUpdateNotification) {
       Position position = Position.fromApi(event.field0);
+
+      position.unrealizedPnl = Amount(_positionService.calculatePnl(position, _price));
```

Comment:
that's a good point, I think the price should really always be the current bid and ask and we should then use the potential closing price accordingly.

I would really like to achieve that the PnL that is displayed is also what the user would actually get if they close with said price. In ItchySats there could be discrepancies because we did not use the correct price in some cases.